DataFormats/Detectors/FIT/FV0/include/DataFormatsFV0/RawEventData.h

```diff
@@ -0,0 +1,243 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+//
+//file RawEventData.h class  for RAW data format
+//Alla.Maevskaya@cern.ch
+// with Artur.Furs
+//
+#ifndef ALICEO2_FIT_RAWEVENTDATA_H_
```

Comment:
Please avoid using the same guard as in the original file: https://github.com/AliceO2Group/AliceO2/blob/6086c834f9715ea587c5cfa1100207fb241b3037/DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/RawEventData.h#L15

Reply:
Good point. Done